src/components/Text/Text.tsx

```diff
@@ -30,6 +30,8 @@ export interface TextProps {
   isLowercase?: boolean;
   /** Optional prop used to remove default spacing */
   noSpace?: boolean;
+  /** Optional prop used to explicitly set the ARIA role */
+  role?: string;
```

Comment:
`role` is an HTML attribute that should be passed by default by Chakra. It is most likely in the `...rest` spread prop value. I'm assuming we want to explicitly make this a prop for devs to know that it _should_ be updated?

Reply:
I added this explicitly because VS Code was saying the `role` prop was not there by default and wasit was throwing a typescript error.  I would rather not include, but this was the only way I knew to get around the error.  Is there another option?

Reply:
@EdwinGuzman Can you suggest another way of doing this?  Normally you don't need to set a role, as it should be an assumed property for `<p>` elements, but for some reason that fundamental `role` is not getting picked up by the a11y tests.  It only passes when it is explicitly set.

Reply:
I think there might be two issues here. The first is that we need to set the type correctly so Typescript knows that it's a native HTML attribute. The other is that `"paragraph"` is not in the list of role values. That is probably why it's not getting picked up by a11y test. I am looking here https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles

I'm taking a closer look at the typescript issue but maybe it's fine to keep for now as an explicit prop.

Reply:
OK, thanks.  For some clarity, adding `role="paragraph"` was something that Clare had suggested and I know she did do some research on that.